numpy/core/code_generators/genapi.py

```diff
@@ -259,7 +259,8 @@ def find_functions(filename, tag='API'):
             elif state == STATE_ARGS:
                 if line.startswith('{'):
                     # finished
-                    fargs_str = ' '.join(function_args).rstrip(' )')
+                    # remove any white space and the closing bracket:
+                    fargs_str = ' '.join(function_args).rstrip()[:-1].rstrip()
```

Comment:
This is necessary, because otherwise it parsing `api_function(..., type NPY_UNUUSED(parameter))` fails because it strips both closing brackets.